### PR TITLE
SW-0911 Issue Resolved

### PR DESCRIPTION
**Pull request by anum-tariq-butt**

### CODE REVIEW

File: src/main/java/org/example/b/Greeting.java [hunk 1]
```diff
@@ -3,8 +3,23 @@
 import static org.example.b.Message.message;
 
 public class Greeting {
+    static int a = 4;
+    static int b;
+
+    static {
+        System.out.println("I'm in static block");
```
Comment (Ahmad-younas) [inline src/main/java/org/example/b/Greeting.java:10]: remove this SOUT command

File: src/main/java/org/example/b/Greeting.java [hunk 2]
```diff
@@ -3,8 +3,23 @@
...
+    }
     public static void main(String[] args) {
         System.out.println("Hello world");
+        Greeting greeting = new Greeting();
+        System.out.println(greeting.a + " " + greeting.b);
+        Greeting.b += 3;
+        System.out.println(greeting.a + " " + greeting.b);
+
+        Greeting greeting1 = new Greeting();
+        System.out.println(greeting1.a + " " + greeting1.b);
```
Comment (Ahmad-younas) [inline src/main/java/org/example/b/Greeting.java:22]: use logger instead of sout

**Review by Ahmad-younas — CHANGES_REQUESTED** [review]
please incorporate comments

File: src/main/java/org/example/b/Greeting.java (hunk 1 above)
Comment (anum-tariq-butt) [reply src/main/java/org/example/b/Greeting.java:10]: done

File: src/main/java/org/example/b/Greeting.java (hunk 2 above)
Comment (anum-tariq-butt) [reply src/main/java/org/example/b/Greeting.java:22]: will do in next PR

**Review by Ahmad-younas — APPROVED** [review]

**Pull request merged**